icu4c/source/common/wintz.cpp

```diff
@@ -13,14 +13,12 @@
 
 #include "unicode/utypes.h"
 
-// This file contains only desktop Windows behavior
-// Windows UWP calls Windows::Globalization directly, so this isn't needed there.
-#if U_PLATFORM_USES_ONLY_WIN32_API && (U_PLATFORM_HAS_WINUWP_API == 0)
+#if U_PLATFORM_USES_ONLY_WIN32_API || U_PLATFORM_HAS_WINUWP_API
```

Comment:
>f U_PLATFORM_USES_ONLY_WIN32_API || U_PLATFORM_HAS_WINUWP_API [](start = 2, length = 61)

Doesn't this also require changes elsewhere, specifically in the caller? putil.cpp has this, which seems like it is wrong now:

#if U_PLATFORM_HAS_WINUWP_API == 0
#   include "wintz.h"
#else // U_PLATFORM_HAS_WINUWP_API
typedef PVOID LPMSG; // TODO: figure out how to get rid of this typedef
#include <Windows.Globalization.h>
#include <windows.system.userprofile.h>
#include <wrl/wrappers/corewrappers.h>
#include <wrl/client.h>

using namespace ABI::Windows::Foundation;
using namespace Microsoft::WRL;
using namespace Microsoft::WRL::Wrappers;
#endif

And then of course the actual call site.

Reply:
The new commits fixed the related codes for this change.